proposals/2022-09-14-kubernetes-versioning.md

```diff
@@ -0,0 +1,208 @@
+# Wasm Shim Versioning in Kubernetes
+Wasm workloads in Kubernetes will likely need to lock into a specific version of a Wasm runtime. This proposal is intended to address this problem.
```

Comment:
How specific? Major? Major.Minor? More specific than that?

Reply:
I believe the problem statement is expanded upon directly after this statement. Should I also add more here?

Reply:
Yeah, I think you probably want to state from the beginning that based on the standard forward-moving velocity of wasm at present (fast-moving!) we want to be able to flexibly specify version requirements from major to minor, and up to and including the exact, specific version (e.g., exactly `v1.0.0-alpha.8e24ee0`).

Also you could make a note that being able to declare such specifity will unlock test and A/B scenarios during a period where the ecosystem is so rapidly evolving.